modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Checkbox/__snapshots__/Checkbox.es.js.snap

```diff
@@ -35,8 +35,7 @@ exports[`Field Checkbox has a helptext 1`] = `
       Type something
     </span>
     <span
-      aria-hidden="false"
```

Comment:
@antonio-ortega, can you please create a single commit to the snapshots regeneration?

Thanks.